API/Modules.md

```diff
@@ -14,12 +14,12 @@ The best place to find API documentation for a Tessel module is through the link
 
 ## Quick links
 
-* [See all the modules and find their API documentation](//tessel.io/modules)
-* [Learn how to use modules on the start page](//start.tessel.io/modules)
+* [See all the modules and find their API documentation](https://tessel.io/modules)
```

Comment:
Oh I found the links being generated incorrectly and saw that explicit http or https would fix that. 

<img width="945" alt="screen shot 2016-06-28 at 9 49 13 am" src="https://cloud.githubusercontent.com/assets/3051193/16417903/b43be91e-3d15-11e6-8acc-f996973f6dad.png">